geomesa-gs-plugin/geomesa-accumulo-gs-plugin/README.md

```diff
@@ -1,11 +1,33 @@
-# GeoMesa Plugin
+# GeoMesa Accumulo GeoServer Plugin
 
-### Building Instructions
+### Installation Instructions
 
-If you wish to build this project separately, you can with mvn clean install
+After building, unzip `target/geomesa-accumulo-gs-plugin-<version>-install.zip` into GeoServer's
+WEB-INF/lib directory.
 
-```geomesa/geomesa-gs-plugin/geomesa-accumulo-gs-plugin> mvn clean install```
+### Additional Jars
 
-This will produce a zip of jars in target/ which are intended to be used as a plugin for Geoserver.  They should be extracted and deployed to
-Geoserver's WEB-INF/lib directory.
+You will also need to copy the required Accumulo and Hadoop jars. The exact jars will vary
+depending on your environment. Some common configurations:
 
+#### Accumulo 1.5
+
+| jar | size |
+| --- | ---- |
+| accumulo-core-1.5.2.jar | 3748459 |
+| accumulo-fate-1.5.2.jar | 99782 |
+| accumulo-start-1.5.2.jar | 53902 |
+| accumulo-trace-1.5.2.jar | 116904 |
+| commons-configuration-1.6.jar | 298829 |
+| libthrift-0.9.0.jar | 347531 |
+| zookeeper-3.3.6.jar | 608239 |
+
+#### Hadoop 2.2
+
+| jar | size |
+| --- | ---- |
+| hadoop-annotations-2.2.0.jar | 16778 |
+| hadoop-auth-2.2.0.jar | 49750 |
+| hadoop-client-2.2.0.jar | 2559 |
+| hadoop-common-2.2.0.jar | 2735584 |
+| hadoop-hdfs-2.2.0.jar | 5242252 |
```

Comment:
Yeah, those were for the index job you could kick off - but we've removed that, so I'm pretty sure they're not necessary anymore.